diagnostic.md

```diff
@@ -3,17 +3,20 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
+<!-- #readme > article
+#readme > article > div
+#user-content-license
+#js-repo-pjax-container -->
+
 
-<!-- your answer ends here -->
 
 ## Question 2
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
+<!-- $("li").find("*") -->
```

Comment:
This will select all `<li>` elements and then find every descendent of every `<li>` element.